### ISSUE TITLE
Add spell checker dictionaries

### ISSUE THREAD

**Issue by LeandroStanger** [flathub/com.wps.Office#5]

**Comment by gasinvein** [comment]
Please try out this build https://github.com/flathub/com.wps.Office/pull/6#issuecomment-457836319

**Comment by gasinvein** [comment]
Good if #6 works for you, but it's better to keep this issue open until the branch with dictionaries is merged to master.

**Comment by toMeloos** [comment]
This is kind of a big deal for everyone using other languages than just English and the solution is already here. Can anyone give that pull request some love and get it merged?

**Comment by gasinvein** [comment]
Closed by #45. Now you can add spellcheck dictionaries as flatpak extensions for the WPS Office app.

**Comment by LeandroStanger** [comment]
flathubbot: FAILURE
https://flathub.org/builds/#/builders/32/builds/7294
```
Publishing build https://hub.flathub.org/api/v1/build/6991
Api call to https://hub.flathub.org/api/v1/build/6991/publish failed with status 400, details: {'current-state': 'published',
```

**Comment by gasinvein** [comment]
@LeandroStanger this doesn't look like an actual error, since
> {'current-state': 'published', 'error-type': 'wrong-published-state', 'expected-state': 'unpublished', 'message': 'Build has already been published', 'status': 400}
Will see if it'll got updated on Flathub later.

And it doesn't prevent you from creating spellcheck dictionaries extensions you need anyway ;)

**Comment by toMeloos** [comment]
> Closed by #45. Now you can add spellcheck dictionaries as flatpak extensions for the WPS Office app.

Deploying the upgrade numbered 11.1.0.8722. Not sure if that's the latest and includes this. Running Fedora Silverblue in Dutch but WPS Writer only showed me WPS's default English and Chinese.

Where are the spellcheck dictionary extension flatpaks themselves? Are these general platform flatpaks or WPS-specific? Are they available on Flathub or do we need to build them?

**Comment by gasinvein** [comment]
@toMeloos There is no dictionary extensions yet, but you can create one yourself and submit it to Flathub. Have a look at the [locales](https://github.com/flathub/com.wps.Office/tree/master/locales) directory in this repo, I've put some stuff there to automate this process as much as possible.
